docs/faq.md

```diff
@@ -200,9 +202,46 @@ kubectl patch serviceaccount default \
 
 See more at [Using Google Container Registry (GCR) with Minikube](https://ryaneschinger.com/blog/using-google-container-registry-gcr-with-minikube/).
 
+### How do I configure a proxy?
+
+Jib currently requires configuring your build tool to use the appropriate [Java networking properties](https://docs.oracle.com/javase/8/docs/api/java/net/doc-files/net-properties.html) (`https.proxyHost`, `https.proxyPort`, `https.proxyUser`, `https.proxyPassword`).
+
+### How can I diagnose problems pulling or pushing from remote registries?
+
+There are a few reasons why Jib may be unable to connect to a remote registry, including:
+
+   - **Access requires a proxy.** See [_How do I configure a proxy?_](#how-do-i-configure-a-proxy) for details.
+   - **The registry does not support HTTPS or its certificates are not configured properly.**  We have a separate document on [handling registries that use self-signed certificates](self_sign_cert.md).  We support an  `allowInsecureRegistries` flag to ignore SSL certificate validation, but we do recommend its use (_Jib 0.9.8_). Finally we do not pass authentication details on non-HTTPS connections, but this can be overridden with the `sendCredentialsOverHttp` system property (_Jib 0.9.8_).
```

Comment:
> The registry does not support HTTPS or its certificates are not configured properly.

I'm wondering naybe we should break this down, because what "The registry does not support HTTPS" sounds to me is that it includes the case of an HTTP server, and in this case, "handling registries that use self-signed certificates" does not apply. Also, about "its certificates are not configured properly", it doesn't mean the certificates are always self-signed. Self-signed certificates are one such case, so if the server indeed has a wrong or expired certificate, installing such certificates to a local trusted CA store does not seem like a good idea. Moreover, although not sure, but especially if the certificate is expired, it may never succeed even after installing it locally?

> we do recommend its use (_Jib 0.9.8_)

It's not clear what "its use" means: installing the self-signed certificates or using `allowInsecureRegistries`? If recommending the latter, I'd move it to the top.

Reply:
I'll separate this into SSL-related and HTTP-related.

_Its_ refers to the `allowInsecureRegistries`.